src/Math-AutomaticDifferenciation/PMDualNumber.class.st

```diff
@@ -244,6 +275,11 @@ PMDualNumber >> raisedToInteger: anInteger [
 		ifFalse: [ self class value: (value raisedToInteger: anInteger) eps: anInteger * eps * (value raisedToInteger: anInteger - 1) ]
 ]
 
+{ #category : #converting }
+PMDualNumber >> real [
+	^ self class value: self value real eps: self eps asComplex real
+]
+
 { #category : #arithmetic }
 PMDualNumber >> reciprocal [
 ^self class value: 1/value  eps: eps /value squared negated
```

Comment:
If it's safe, perhaps indent this line too.

Reply:
what do you mean ?

Reply:
I mean to correct the spacing so that it looks like
```
PMDualNumber >> reciprocal [
       ^self ...
```

Reply:
ok